docs/en/global-administration.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Global Administration
+
+contributors: [nakata5321]
+translated: true
+---
+
+This article will provide you to how to set up a new user to your Home Assistant.
+
+## Use DApp
+
+Now it's time to create a new user for your Home Assistant. People, who uses Home Assistant, shouldn't use `SUB_OWNER` or `SUB_CONTROLLER` accounts, because,
```

Comment:
> Now it's time to create a new user for your Home Assistant

 явно из прошлой версии, не та фраза для начала статьи. Можно типа, 

```suggestion
Home Assistant users shouldn't take `SUB_OWNER` or `SUB_CONTROLLER` accounts, because,
```